hw/dv/tools/gen_ral_pkg.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
@imphil 
I whipped up a quick script to manually generate the fusesoc core file in the scratch space where I generate the register model for DV as well (see template file above). Fusesoc generator probably is a better solution, but I am not sure I understood what needed to be done from the documentation and if it supported the core generation in a scrach directory path that is controlled externally.